---
title: "An Lab - Code"
layout: textlay
excerpt: "An Lab -- Code."
sitemap: false
permalink: /code/
---


# Code & Dataset

---

## Brain Transcriptome Single-cell (BTS) Atlas


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/SCN2A_multiplot.png" style="width: 300px">

Brain Transcriptome Single-cell Atlas (BTS) Anndata, Seurat object, and Celltypist model for further use of the atlas. The Celltypist model can be utilized to accurately annotate cell types in new datasets based on the atlas. Plots illustrating the expression profile for 3,380 neurological disorder risk genes across the atlas are also uploaded. Further availability for the data can be requested by the corresponding author.

This dataset is published in Kim, S., Lee, J., Koh, I.G. et al. An integrative single-cell atlas for exploring the cellular and temporal specificity of genes related to neurological disorders during human brain development. Exp Mol Med 56, 2271–2282 (2024). https://doi.org/10.1038/s12276-024-01328-6


- <a href="https://zenodo.org/records/14177002"><i class='fa fa-database'></i> Source</a>
- <a href="https://bts-brain-cell-atlas.cells.ucsc.edu/"><i class='fa fa-map'></i> UCSC Cell Browser</a>
- <a href="https://www.nature.com/articles/s12276-024-01328-6"><i class='fa fa-book'></i> Paper</a>

## CWAS-Plus: A new version of Category-Wide Association Study


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/cwas.png" style="width: 300px">

A new Python package runs category-wide association study with better user-interface. Now you can use this with pip install!


- <a href="https://github.com/joonan-lab/cwas"><i class='fab fa-github'></i> Source</a>
- <a href="https://cwas-plus.readthedocs.io/en/latest/index.html"><i class='fa fa-file-code-o'></i> Manual</a>


## CWAS: Category-Wide Association Study


The Cython wrapper script runs category-wide association study presented in [An et al. (2018) Science](https://www.ncbi.nlm.nih.gov/pubmed/30545852). This is an Amazon Machine Images (AMI) on Amazon Web Services (AWS) Cloud Computing Services.


- <a href="https://github.com/sanderslab/cwas"><i class='fab fa-github'></i> Source</a>
- <a href="https://www.ncbi.nlm.nih.gov/pubmed/30545852"><i class='fa fa-book'></i> Paper</a>

---

## De novo Risk Score and Annotation Clustering

<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/annocl.png" style="width: 300px">

This repository contains R scripts for risk score analysis and annotation clustering in [An et al. (2018) Science](https://www.ncbi.nlm.nih.gov/pubmed/30545852).

- <a href="https://github.com/lingxuez/WGS-Analysis"><i class='fab fa-github'></i> Source</a>
- <a href="https://www.ncbi.nlm.nih.gov/pubmed/29184211"><i class='fa fa-book'></i> Paper</a>

## CatBurden: Category-based burden test


<img src="{{ site.url }}{{ site.baseurl }}/images/codepic/catburden.png" style="width: 300px">

The R/Python wrapper script includes commands for the category-based burden test and the first edition of CWAS in the [Werling et al. (2018) Nature Genetics](https://www.ncbi.nlm.nih.gov/pubmed/29700473). The scripts and materials are distributed in AMI (ID: `ami-1eefb064`) and you can deploy the workflow using AWS. We recommend using the 100 Gib size EBS and the EC2 instance type `m4.large` or `m4.xlarge`.

- <a href="https://github.com/sanderslab/WGS-pipeline"><i class='fab fa-github'></i> Source</a>
- <a href="https://www.ncbi.nlm.nih.gov/pubmed/29700473"><i class='fa fa-book'></i> Paper</a>


## Hail tutorial (Korean)

우리말로 된 Hail 튜토리얼입니다.

- <a href="https://github.com/joonan-lab/hail_tutorial"><i class='fab fa-github'></i> Source</a>
