---
title: "An Lab - Research"
layout: textlay
excerpt: "An Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

---

## Understanding the genetic architecture of autism spectrum disorder

<div style="text-align: justify">
The recent advent of next-generation sequencing technologies has dramatically changed the nature of translational research. Human genetics is no exception â€“ it has never been easier to investigate human patient genomes at the nucleotide level to identify disease-associated mutations. My long-term research interest is to understand genetic heterogeneity underlying autism spectrum disorder (ASD) ([Choi & An 2021](https://www.sciencedirect.com/science/article/pii/S0149763421002700)). As a leading investigator in the largest genomic consortium of autism in the world, I have analyzed whole genome sequencing (WGS) data of 2,000 quartet families with autism and evaluated mutation burden in the noncoding genome contributing to autism ([An et al. 2018](https://www.science.org/doi/abs/10.1126/science.aat6576)). Since I established my research group at Korea, our lab has been studying a complete genetic architecture of Korean ASD families. We have been scrutinizing a wide range of genetic factors, including common, rare and de novo variants, in individuals diagnosed with ASD from Korean families.
</div>


## Characterization regulatory noncoding variant in neurodevelopment and human disorders

<div style="text-align: justify">
We previously proposed a statistical framework, Category-wide association study (CWAS) to evaluate risk noncoding mutation in WGS data of autism families ([Werling et al. 2018](https://www.nature.com/articles/s41588-018-0107-y)). Our lab continues to develop the CWAS method to integrate single cell datasets of brain tissues and chromatin interactions. We are actively improving this method to enhance functionality and usability in various steps, reducing time and computation in category generation and multiple testing. We aim to comprehend our understanding to noncoding association for ASD and neurodevelopmental disorders. In addition, we are applying this method to WGS data of Korean Alzheimer's disease patients and find noncoding mutations underlying risk development. 
</div>


## Integrative multi-omics approaches to understand complex diseases

<div style="text-align: justify">
My long-term research interest is to understand extreme genetic heterogeneity underlying human complex diseases. I have tested the concept that multiple risk genes converge on a smaller number of critical biological processes. We previously developed a prediction model based on computational approach, searching for cohesive biological network in ASD ([An et al. 2014](https://www.nature.com/articles/tp201438)). Integrating this model into in vitro functional characterization has led to delineate key pathways, including axonal guidance and NRXN complex, in autism and evaluated candidate noncoding mutations using functional validation ([Williams et al. 2018](https://www.nature.com/articles/s41380-018-0049-x)). More recently, our lab has integrated large-scale WGS and transcriptomics datasets of human post-mortem cortex from fetal to adult periods and analyzed how genetic variation impact on gene expression in developing cortex ([Werling et al. 2020](https://www.sciencedirect.com/science/article/pii/S2211124720303673)). Our lab has extended our systems approaches to understand the core of disease pathology and to find cancer subtypes and tumor progression ([Heo et al. 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8334347/)). Multi-omics with large-scale proteomics is an emerging field in biomedical science and holds potential to mechanism and translational research. From this end, our lab has been developing analytic frameworks for multi-omics analyses of Korean lung cancer patients, including genomics, transcriptomics, proteomics, phospho-proteomics, and acetyl-proteomics, and characterized cancer subtypes and tumor microenvironment. 
</div>
----
