---
title: "An Lab - Research"
layout: textlay
excerpt: "An Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

---

## Genetic architecture of autism

<div style="text-align: justify">
Our lab is dedicated to studying the genetic heterogeneity underlying autism, with a focus on the distinct genetic architecture observed in East Asian populations. As a member of the autism genomic consortium, I have analyzed large-scale whole genome sequencing data from autism families, examining the mutation burden within noncoding regions of the genome to assess its contribution to autism ([An et al. 2018, Science](https://www.science.org/doi/full/10.1126/science.aat6576), [Werling et al. 2019, Nature Genetics](https://www.nature.com/articles/s41588-018-0107-y), [Williams et al. 2019, Molecular Psychiatry](https://www.nature.com/articles/s41380-018-0049-x)). Since establishing my research group in Korea University in 2019, we have concentrated on exploring a genetic architecture of autism in Korean families. Our investigations encompass a wide range of genetic factors, including common, rare, and de novo variants, which we are analyzing within one of the largest East Asian cohorts for autism. Notably, our research has recently revealed sex-specific patterns in genetic risk factors among Korean autism families, showing that these differences may influence phenotypic severity and familial patterns in autism ([Kim et al. 2024, Genome Medicine](https://link.springer.com/article/10.1186/s13073-024-01385-6)).

Our studies extend into complex regulatory mechanisms, such as short tandem repeat expansions and transcriptional regulation, and their impact on brain development in autism. We identified that rare tandem repeat expansions in cortical layer-specific genes are significantly associated with autism risk and phenotypic severity ([Kim et al. 2024, Psychiatry and Clinical Neurosciences](https://onlinelibrary.wiley.com/doi/full/10.1111/pcn.13676)). In collaborative efforts, we have further demonstrated that ASD-linked transcriptional regulators converge on shared genomic loci in the brain, illuminating regulatory pathways that drive neurodevelopment ([Darbandi et al. 2024, Cell Reports](https://www.cell.com/cell-reports/fulltext/S2211-1247(24)00657-0), [Kim et al. 2024, Experimental & Molecular Medicine](https://www.nature.com/articles/s12276-024-01328-6)). Additionally, our contributions to large-scale genomic analyses have distinguished gene-specific associations within ASD and clarified the role of rare coding variations across neurodevelopmental disorders ([Satterstrom et al. 2020, Cell](https://www.cell.com/cell/fulltext/S0092-8674(19)31398-4)).
</div>


## Deep learning to understand regulatory pattern in noncoding genome

<div style="text-align: justify">
Our lab uses advanced deep learning techniques, including large language models, to study noncoding regulatory mutations associated with autism and other neurodevelopmental disorders. We've developed the CWAS framework ([Kim et al. 2024](https://academic.oup.com/bib/article/25/4/bbae323/7706421)) to analyze whole genome sequencing data from autism families. Currently, we're integrating single-cell multiomics datasets from developing human brains to further refine our approach. By combining computational methods with rich genomic and epigenomic data, we aim to uncover how noncoding mutations contribute to neurodevelopmental risk. Our work continues to evolve as we improve our models and extend our research to cover more neurodevelopmental conditions. Through these efforts, we strive to develop more accurate diagnostic tools and biomarkers for autism and related disorders, potentially improving early detection and intervention strategies.
</div>


## Integrative multi-omics approaches to understand complex disorder

<div style="text-align: justify">
My research focus is centered on exploring the extreme genetic heterogeneity that underlies complex human disorder. I have examined the hypothesis that multiple risk genes converge on a reduced number of crucial biological processes. To this end, I have developed a computational prediction model to identify cohesive biological networks in autism ([An et al. 2014](https://www.nature.com/articles/tp201438)). Further integration of this model with in vitro functional characterization has led to the identification of key pathways, including axonal guidance and the NRXN complex, in autism, which were evaluated through functional validation ([Williams et al. 2018](https://www.nature.com/articles/s41380-018-0049-x)). Our lab has integrated large-scale whole-genome sequencing and transcriptomics datasets of human post-mortem cortex across fetal to adult stages to analyze the impact of genetic variation on gene expression in developing cortex ([Werling et al. 2020](https://www.sciencedirect.com/science/article/pii/S2211124720303673)). Furthermore, we have applied our systems approach to understand the core of pathology and to identify cancer subtypes and tumor progression, as outlined in [Heo et al. 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8334347/). Multi-omics analysis, particularly through large-scale proteomics, is an emerging area in biomedical science and holds great promise for mechanism and translational research. To that end, our lab has been developing analytical frameworks for multi-omics analysis of Korean lung cancer patients, including genomics, transcriptomics, proteomics, phospho-proteomics, and acetyl-proteomics, to characterize cancer subtypes and tumor microenvironment.
</div>
----
