---
title: "An Lab - Research"
layout: textlay
excerpt: "An Lab -- Research"
sitemap: false
permalink: /research/
---

# Research

---

## Understanding the genetic architecture of autism spectrum disorder

<div style="text-align: justify">
The emergence of next-generation sequencing technologies has had a profound impact on the field of translational research, with human genetics being a prime example. It is now possible to examine human patient genomes at the nucleotide level to identify mutations linked to diseases with unprecedented ease. My research focus is centered on understanding the genetic heterogeneity underlying autism spectrum disorder (ASD) ([Choi & An 2021](https://www.sciencedirect.com/science/article/pii/S0149763421002700)). As a member of the genomic consortium for autism, I have analyzed whole genome sequencing (WGS) data from 2,000 quartet families with autism, evaluating the mutation burden in the noncoding genome and its contribution to autism ([An et al. 2018](https://www.science.org/doi/abs/10.1126/science.aat6576)). Since establishing my research group in Korea, we have been exploring the complete genetic architecture of Korean families with ASD. Our lab is examining a broad range of genetic factors, including common, rare, and de novo variants, in individuals diagnosed with ASD from Korean families.
</div>


## Characterization regulatory noncoding variant in neurodevelopment and human disorders

<div style="text-align: justify">
In our previous study ([Werling et al. 2018](https://www.nature.com/articles/s41588-018-0107-y)), we introduced the Category-wide Association Study (CWAS) statistical framework to assess the risk of noncoding mutations in whole genome sequencing (WGS) data of autism families. Our laboratory is continuously refining the CWAS methodology by integrating single-cell datasets of brain tissue and chromatin interactions. Efforts are being made to optimize the functionality and usability of the method, including reducing the time and computational requirements for category generation and multiple testing. Our ultimate goal is to deepen our understanding of the noncoding association for autism spectrum disorder (ASD) and other neurodevelopmental disorders. Furthermore, we are applying the CWAS method to WGS data of Korean Alzheimer's disease patients to identify noncoding mutations that contribute to the risk of disease development.


</div>


## Integrative multi-omics approaches to understand complex diseases

<div style="text-align: justify">
My research focus is centered on exploring the extreme genetic heterogeneity that underlies complex human diseases. I have examined the hypothesis that multiple risk genes converge on a reduced number of crucial biological processes. To this end, I have developed a computational prediction model to identify cohesive biological networks in autism ([An et al. 2014](https://www.nature.com/articles/tp201438)). Further integration of this model with in vitro functional characterization has led to the identification of key pathways, including axonal guidance and the NRXN complex, in autism, which were evaluated through functional validation ([Williams et al. 2018](https://www.nature.com/articles/s41380-018-0049-x)). Our lab has integrated large-scale whole-genome sequencing and transcriptomics datasets of human post-mortem cortex across fetal to adult stages to analyze the impact of genetic variation on gene expression in developing cortex ([Werling et al. 2020](https://www.sciencedirect.com/science/article/pii/S2211124720303673)). Furthermore, we have applied our systems approach to understand the core of disease pathology and to identify cancer subtypes and tumor progression, as outlined in [Heo et al. 2021](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8334347/). Multi-omics analysis, particularly through large-scale proteomics, is an emerging area in biomedical science and holds great promise for mechanism and translational research. To that end, our lab has been developing analytical frameworks for multi-omics analysis of Korean lung cancer patients, including genomics, transcriptomics, proteomics, phospho-proteomics, and acetyl-proteomics, to characterize cancer subtypes and tumor microenvironment.
</div>
----
